bot.py

```diff
@@ -53,15 +55,22 @@
 ]
 
 
-def seconds_until_9am():
+def seconds_until_9am() -> float:
     now = datetime.datetime.now()
     target = (now + datetime.timedelta(days=1)).replace(hour=9, minute=0, second=0, microsecond=0)
     diff = (target - now).total_seconds()
     print(f"{target} - {now} = {diff}")
     return diff
 
+condslist = [
+    dt.Condition("Darren is never wrong, respect your old GMs", dt.Contains("darren")),
```

Comment:
nitpick. But I think it reads better if we have the conditions first, then the result. 
```
    dt.Condition(dt.Contains("darren"), "Darren is never wrong, respect your old GMs"),
```